IOMC/ParticleGuns/plugins/BuildFile.xml

```diff
@@ -0,0 +1,10 @@
+<library   name="IOMCParticleGunsFlatProtonLogKsiLogTGunPlugins" file="*.cc">
```

Comment:
Is this file needed at all ? It seems the `plugins` directory doesn't exists here: https://github.com/cms-sw/cmssw/tree/CMSSW_8_1_X/IOMC/ParticleGuns and other guns seems to work.